autogen/agenteval/agent_eval.py

```diff
@@ -0,0 +1,58 @@
+from criterion import Criterion
```

Comment:
Let's discuss should we create a folder under "autogen" or under "autogen/agentchat/contrib". I personally think it is fine to put it under "autogen" directly, but other people might have different opinions. There was a debate on where "AgentOptimizer" should be placed early this morning, and we should be consistent with its final decision.

Reply:
@jluey1 @BeibinLi  great point! let's make a decision about it

Reply:
My instinct is for a separate folder but I'm happy to follow whatever convention AgentOptimizer establishes.

Reply:
@BeibinLi was there any decision ever made about folder structure?

Reply:
going to leave as-is for now